src/pages/Message.tsx

```diff
@@ -0,0 +1,73 @@
+import MessageContent from "../components/Message/MessageContent";
+import MessageListItem from "../components/Message/MessageListItem";
+
+const TEMP_USER: User = {
+  role: "Regular",
+  isOnline: false,
+  posts: [],
+  likes: [],
+  comments: [],
+  followers: [],
+  following: [],
+  notifications: [],
+  messages: [],
+  _id: "675c637f7812b71a3cef02b5",
+  fullName: "하치와레",
+  email: "abc2@gmail.com",
+  createdAt: "2024-12-13T16:40:31.755Z",
+  updatedAt: "2024-12-16T03:29:53.507Z",
+  image:
+    "https://res.cloudinary.com/learnprogrammers/image/upload/v1734108092/user/6559338b-237f-43f1-bf6d-764b9e133cf4.jpg",
+  coverImage: "",
+};
+
+export default function Message() {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      <section className="flex flex-row">
+        <article className="flex flex-col justify-between w-[721px] ">
+          <div className="px-10">
+            <div className="flex items-center mt-[38px] mb-[29px]">
+              <img
+                src={TEMP_USER.image || "/logo.png"}
+                alt="프로필 이미지"
+                className="w-[30px] h-[30px] mr-3 mb-1 rounded-full"
+              />
+              <p className="text-lg dark:text-white">
+                닉네임{/* {user?.fullName} */}
+              </p>
+            </div>
+            <MessageContent
+              isOutgoingMessage={false}
+              message="안녕하세요~~~~ 반갑습니다아~~~~ ^_^"
+            />
+          </div>
+          <div className="border-t border-gray-c8 px-10 pb-8">
+            <form className="flex flex-col" onSubmit={handleSubmit}>
+              <textarea
+                className="w-full mt-5 p-3 border border-gray-c8 rounded-[15px] whitespace-pre-wrap focus:text-gray-22 placeholder:text-[#cecece]"
+                placeholder="메세지를 입력해주세요."
+              />
+              <button
+                className="self-end w-20 h-10 rounded-[15px] mt-4 py-[7px] px-[19px] bg-primary"
```

Comment:
```suggestion
                className="self-end w-20 h-10 rounded-[15px] mt-4 py-[7px] px-[19px] bg-primary text-gray-22"
```
위에서 글자색 기본으로 #222로 지정되어 있다고 했는데 여기서는 왜 지정하냐면요!
#222로 지정된 텍스트는 다크모드에서 자동으로 #eee로 적용되어 버립니다 (저희가 설정을 그렇게 했어요)

하지만 버튼 색상은 동일한데 텍스트가 #eee로 보이면 노란 버튼에서는 텍스트가 안보이겠죠!
그래서 여기서는 dark모드에서는 변경되지 말라고 직접 지정하는 것입니당